core/MyGatewayTransportMQTTClient.cpp

```diff
@@ -159,7 +159,11 @@ bool reconnectMQTT(void)
 
 		return true;
 	}
+#if defined(MY_MQTT_ETH_CLIENT_CONNECTION_TIMEOUT)
```

Comment:
How about setting the default value in MyConfig.h (like we do for baud rate [here](https://github.com/mysensors/MySensors/blob/51742c95aa03c04ee41925baed56b665eb9793a2/MyConfig.h#L162)). That way we don't need to have the if clause here and on line 258.

Reply:
As you wish, no problem! Will do in an hour

Reply:
done